docs/howto/debug.md

```diff
@@ -2,6 +2,13 @@
 
 In this section, you will learn how to debug the compilation process easily and find help in the case that you cannot resolve your issue.
 
+## Compiler debug and verbose modes
+
+There are two [configuration](../howto/configure.md) options that you can use to understand what's happening under the hood during compilation process.
+
+- **compiler_verbose_mode** will print the passes applied by the compiler and let you see the transformations done by the compiler. Also, in case of crashes, it could narrow down the crash location. 
+- **compiler_debug_mode** is a lot more detailed version of the verbose mode. Even better for crashes.
+
```

Comment:
thanks a lot @umut-sahin ! could you maybe mention that for jupyter/notebooks (I don't know exactly but you do), it will not work well? if ever people have issues in jupyter

Reply:
Sure, done!

Reply:
tx